docs/jwst/tso_photometry/reference_files.rst

```diff
@@ -1,27 +1,38 @@
-Reference File
-==============
-The tso_photometry step uses a TsoPhotModel reference file, reference type
-TSOPHOT, that supplies values of radius (in pixels) for the target aperture
-and the inner and outer radii for the background annulus.
+Reference Files
+===============
+The ``tso_photometry`` step uses a TSOPHOT reference file.
 
-.. include:: ../includes/standard_keywords.rst
+TSOPHOT Reference File
+----------------------
+
+:REFTYPE: TSOPHOT
+:Data model: `~jwst.datamodels.TsoPhotModel`
+
+The TSOPHOT reference file contains photometry aperture radii for
+TSO imaging observation modes.
 
 .. include:: tsophot_selection.rst
 
+.. include:: ../includes/standard_keywords.rst
+
 Type Specific Keywords for TSOPHOT
-----------------------------------
-These keywords are required to be present in a TsoPhotModel reference file.
-
-========  ====================
-Keyword   Model Name
-========  ====================
-EXP_TYPE  meta.exposure.type
-TSOVISIT  meta.visit.tsovisit
-========  ====================
-
-TSOPHOT Reference File Format
------------------------------
-TSOPHOT reference files are ASDF files.  An object called 'radii' in a
+++++++++++++++++++++++++++++++++++
+In addition to the standard reference file keywords listed above,
+the following keywords are *required* in TSOPHOT reference files,
+because they are used as CRDS selectors
+(see :ref:`tsophot_selectors`):
+
+=========  =========================
+Keyword    Data Model Name
+=========  =========================
+EXP_TYPE   model.meta.exposure.type
+TSOVISIT   model.meta.visit.tsovisit
+=========  =========================
+
+Reference File Format
++++++++++++++++++++++
+TSOPHOT reference files are ASDF format.
+An object called 'radii' in a
```

Comment:
This looks like an errant sentence fragment.

Reply:
No, it's just the first part of the larger sentence "An object called 'radii' in a TSOPHOT file defines the radii that the step needs." It just got split in a funny way across lines.